analyticConfigurations/metric_meta-aws.ecs.service.json

```diff
@@ -2,14 +2,22 @@
   "analyticConfiguration": {
     "metrics": [
       {
-        "match": "^aws\\.ecs\\.(cpu.*|memory.*)",
+        "match": "^aws\\.ecs\\.memory.*",
```

Comment:
I think memory will have this issue as well. Some of our services run over 100%.